lib/iris/fileformats/netcdf.py

```diff
@@ -35,8 +35,7 @@
 import warnings
 
 import biggus
-import iris.proxy
-iris.proxy.apply_proxy('netCDF4', globals())
+import netCDF4
```

Comment:
`netCDF4` isn't optional (e.g. we list it in `minimal-conda-requirements.txt`), hence I've just made it a regular import.